flow/project.py

```diff
@@ -2476,7 +2609,7 @@ def select(operation):
                         for job in jobs:
                             operations.extend(
                                 flow_group._create_run_job_operations(
-                                    self._entrypoint, default_directives, job, ignore_conditions))
+                                    self._entrypoint, default_directives, [job], ignore_conditions))
```

Comment:
I dont understand why we need to pass [job] in as a list? Would this be better if handled in `_create_run_job_operations()`?

Reply:
The plan is to pass in a single aggregate (which is a list of jobs) to every internal method. Since we're currently dealing with aggregates of 1 in this PR and all the internal methods (of `FlowGroup`) now take a list of jobs, I thought that to maintain consistency, we should to pass `[job]` instead of `job`.